service/debugger/debugger.go

```diff
@@ -557,14 +557,59 @@ func (d *Debugger) ClearBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoint
 	d.processMutex.Lock()
 	defer d.processMutex.Unlock()
 
-	var clearedBp *api.Breakpoint
-	bp, err := d.target.ClearBreakpoint(requestedBp.Addr)
+	var bps []*proc.Breakpoint
+	var remaddrs []string
+	var errs []error
+	addrnum := 0
+
+	clear := func(addr uint64) {
+		addrnum++
+		bp, err := d.target.ClearBreakpoint(addr)
+		if err != nil {
+			remaddrs = append(remaddrs, fmt.Sprintf("%d", addr))
```

Comment:
Instead of printing the remaining addresses, could we remove this variable altogether and make better use of the error messages returned to inform the user about what went wrong?

Reply:
The error output should have enough context to inform the user what's gone wrong. So instead of collecting all failed addresses and presenting them to the user, can we just present them err.Error() instead for each error that failed? So instead of just showing the first error, maybe print something like (psuedocode): `"the following error(s) occurred when clearing the breakpoint at <addr>"; for (error in errors) { println(error.Error()) }`

Reply:
How about this?